components/dialog/dialog.js

```diff
@@ -128,7 +157,7 @@ class Dialog extends LocalizeCoreElement(AsyncContainerMixin(DialogMixin(LitElem
 
 		if (!this._titleId) this._titleId = getUniqueId();
 		const inner = html`
-			<div class="d2l-dialog-inner">
+			<div class="d2l-dialog-inner"  style=${styleMap(heightOverride)}>
```

Comment:
I had to override the dialog-inner class rather than the outer class because the d2l-dialog-outer class is controlled by the dialogMixin. I didn't want to add the height override there, as the mixin would then have to know about the fullscreen-specific styling and dialog mobile styles. I decided to keep the mixin generic and keep the component-specific styling here.